Readme.md

```diff
@@ -1,3 +1,7 @@
+## Motivation
+Avoid git errors like "git checkout failed" for file names when pulling from a MS Windows client.
+File name that are invalid for MS Windows may have been check in under Linux.
```

Comment:
"File name" -> "File names"
"check in" -> "checked in"

Reply:
Also "a MS" -> "an MS"?